### ISSUE TITLE
some question about softmax-splatting

### ISSUE THREAD

**Issue by LINSEVENQI** [sniklaus/pytorch-sepconv#47]
Hello, what should I do if I want to use your softmax-splatting network to test an N-frame video?

**Comment by sniklaus** [comment]
Thank you for your interest in our Softmax Splatting paper! I am afraid that you are creating this issue in the wrong Github repository. The repository for our most recent work on Softmax Splatting for video frame interpolation is here: https://github.com/sniklaus/softmax-splatting

I just noticed that I accidentally disabled the issues feature for that repository, which is why you probably posted it here instead. My apologies, I enabled it again.

Coming back to your initial question, I am afraid that this repository only includes the code for the softmax splatting operator and not the entire video frame interpolation pipeline. You will thus have to go off of the metrics presented in our paper for quantitative comparisons, we also show all results for the low-res benchmarks on the project website: http://sniklaus.com/softsplat-results
